Sources/MapboxMaps/Gestures/GestureManager.swift

```diff
@@ -101,6 +101,10 @@ public final class GestureManager: NSObject {
             requireGestureToFail(allowedGesture: pitchHandler, failableGesture: panHandler)
         }
 
+        if let pinchHandler = gestureHandlers[.pinch], let panHandler = gestureHandlers[.pan] {
+            requireGestureToFail(allowedGesture: pinchHandler, failableGesture: panHandler)
```

Comment:
I always have to look up the implementation of this method to remember what it does. Here's the implementation:

```
    internal func requireGestureToFail(allowedGesture: GestureHandler, failableGesture: GestureHandler) {
        guard let failableGesture = failableGesture.gestureRecognizer else { return }
        allowedGesture.gestureRecognizer?.require(toFail: failableGesture)
    }
```

A couple thoughts:
1. This should be a method on GestureHandler `func require(toFail other: GestureHandler)`
2. Is it necessary for `GestureHandler.gestureRecognizer` to be optional? If not, let's just delete this method entirely and use the underlying UIGestureRecognizer method directly.

Reply:
I will circle around with folks to try and understand why these were made optional in the first place. I don't want to make that change without understanding the reason. I agree with this assessment but I'm reluctant to make the change without knowing the original intention here.

Also, this feels like it's starting to creep into some fundamental API changes around gestures that several people have acknowledged. How far do we want to take this under this PR? Is there an opportunity for follow up work here to address the big picture?